app/main.py

```diff
@@ -4,4 +1,31 @@
+import requests
+import os
+
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+city = "Paris"
+
+
+def get_weather(api_key: str) -> None:
+    weather_paris_url = f"{BASE_URL}?key={api_key}&q={city}"
+    try:
+        response = requests.get(weather_paris_url)
+    except requests.exceptions.ConnectionError:
+        print("Connection Error")
+
+    weather = response.json()
+
+    if response.status_code == 200:
+        print(
+            f"{weather["location"]["name"]}/"
+            f"{weather["location"]["country"]} "
+            f"{weather["location"]["localtime"]} "
+            f"Weather: {weather["current"]["temp_c"]} "
+            f"Celsius, {weather["current"]["condition"]["text"]}"
+        )
+    else:
+        "There is no data about weather"
```

Comment:
This line doesn't do anything. It seems like you wanted to print a message when there's no data about the weather. However, you're just creating a string and not doing anything with it. If you want to print it, you need to use the `print` function.